### DISCUSSION TITLE
Thoughts on adding input fields to ColorPicker?

### DISCUSSION THREAD

**Discussion by TKDev7** [mantinedev/mantine#390] [Feedback]
Like this:

![image](https://user-images.githubusercontent.com/30705623/140618912-88c6b5c1-6515-427b-b0c6-c9b7591368b0.png)

**Comment by Paraxdev** [comment]
I would like to have that too!

**Comment by rtivital** [comment]
Sounds reasonable, we will add this in 3.2 or 3.3 release
